tutorials/rendering/jitter_stutter.rst

```diff
@@ -147,6 +147,224 @@ iOS
+
+    .. image:: img/windows-no-boost.webp
+
+For NVIDIA on Windows, set "Power management mode" on "Prefer maximum performance".
+
+.. image:: img/nv-power-profile.webp
+
+**Solution (Linux):** Set the CPU governor to Performance:
+
+.. code:: sh
+
+    sudo cpupower frequency-set -g performance
+
+    # AMD
+    echo high | sudo tee /sys/class/drm/card1/device/power_dpm_force_performance_level
+
+You can try `CoreCtrl <https://gitlab.com/corectrl/corectrl>`__ or `Feral GameMode <https://github.com/FeralInteractive/gamemode>`__
+for user-friendly GUI to control power profiles for individual applications.
+
+On NVIDIA, open **nvidia-settings** and go to **GPU 0 > PowerMizer > Preferred Mode: Prefer Maximum Performance**.
+
+.. image:: img/nv-linux-powermizer.webp
+
+Windows: Unlicensed
+~~~~~~~~~~~~~~~~~~~
+
+First of all, "unactivated" or "unlicensed" Windows can have arbitrary limitations imposed by Microsoft.
+These limitations can change over time and an important one is that Microsoft may decide to restrict
+applications from entering "Hardware Independent Flip" modes which provides the lowest latency experience
+while in fullscreen.
+
+Second, even if Microsoft decides to not artificially limit "Hardware Independent Flip"; if
+you're seeing this watermark:
+
+.. image:: img/activate-windows.webp
+
+Chances are you're not running in "Hardware Independent Flip" because Windows must use DWM
+compositing to display the watermark on top of Godot. And DWM Compositing prevents entering the desired mode.
+
+**Solution:** Activate Windows by buying an original license key.
+
+Windows: Presentation Mode
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Windows has many ways of presenting to the screen. `Special K has an in-depth explanation of them <https://wiki.special-k.info/Presentation_Model>`__
+
+But we can basically reduce them to the following:
+
+**Good**
+
+1. Hardware: Independent Flip (also called "iFlip").
+2. Hardware: Legacy Flip.
+3. Hardware Composed: Independent Flip.
+
+**Bad**
+
+1. Composed: Flip.
+
+You can use `PresentMon <https://github.com/GameTechDev/PresentMon/releases>`__ to see which mode Godot is in.
+While in Fullscreen, Godot should be in one of the "good" ones.
+
+If Godot is not in one of the "bad" ones, there may be an overlay that is causing problems
+(e.g. AMD Overlay, NVIDIA's overlay, Microsoft Game Bar, "Activate Windows" watermark, etc).
+
+Godot can only enter the good ones if it's fully covering a single monitor with nothing on top.
+Some newer HW may be able to enter "Hardware Composed: Independent Flip" even if not fullscreen though.
+
+.. warning::
+
+    **Set PresentMon to windowed mode**. Otherwise PresentMon's overlay `ironically prevents apps <https://github.com/GameTechDev/PresentMon/issues/367>`__
+    from entering the "good" modes as it is displayed on top of Godot.
+    You will have to use the recording option and later see the CSV capture to see what mode Godot was in.
```

Comment:
```suggestion
    **Set PresentMon to windowed mode**. Otherwise, PresentMon's overlay
    `ironically prevents apps <https://github.com/GameTechDev/PresentMon/issues/367>`__
    from entering the "good" modes as it is displayed on top of Godot.
    You will have to use the recording option and later see the CSV capture
    to see what mode Godot was in.
```